src/views/UnauthenticatedApp/__tests__/UnauthenticatedApp.test.tsx

```
@@ -0,0 +1,192 @@
+  return (
+    <div>
+      <h2 data-testid="setup-title">{mockI18n.t('setup')}</h2>
+      <div>
+        <span data-testid="language-label">{mockI18n.t('language')}:</span>
+        {Object.entries(languages).map(([key, obj]) => (
+          <button
+            key={key}
+            onClick={() => mockI18n.changeLanguage(key)}
+            disabled={mockI18n.resolvedLanguage === key}
+            data-testid={`language-${key}`}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)